vulcan/lib/server/workflows/scripts/magma_retrieve_paths.py

```diff
@@ -0,0 +1,47 @@
+from archimedes.functions.dataflow import output_tsv, input_var
+from archimedes.functions.magby import Magby
+
+input_records = input_var('record_ids').split(", ")
+token = input_var('token')
```

Comment:
I think this should be an environment variable in the future, too? Like `project_name` and `magma host`

Reply:
Agreed.  I moved this around after, but have that noted in the comments in lines 11&12...

How to implement environment variables seems a discussion for the larger group, perhaps in standup?